myhpi/templates/footer.html

```diff
@@ -12,9 +12,16 @@
     {% endfor %}
     <div class="footer-category">
       <h2>{% translate "Language" %}</h2>
-        {% for translation in page.get_translations %}
-            <a href="{{ translation.url }}">{{ translation.locale }}</a>
-        {% endfor %}
+      {% for translation in page.get_translations %}
+      <form id="language-form-{{ translation.locale }}" action="{% url 'set_language' %}" method="post" style="display: none;">
+        {% csrf_token %}
+        <input name="next" type="hidden" value="{{ redirect_to }}">
+        <input type="hidden" name="language" value="{{ translation.locale.language_code }}">
+      </form>
+      <a href="#" onclick="document.getElementById('language-form-{{ translation.locale }}').submit(); return false;">
```

Comment:
I thin I'd prefer using a button with the `btn-link` class to style it as a link so that it works without js